cmd/bundle/main.go

```diff
@@ -96,6 +106,43 @@ func Do(ctx context.Context) error {
 	}
 
 	log.Printf("Image content was extracted to %s\n", flagValues.target)
+	if flagValues.showListing {
+		t := tablewriter.NewWriter(os.Stdout)
+		t.SetBorder(false)
+		t.SetColumnSeparator("")
+		t.SetAutoWrapText(false)
+
+		// ignore any errors when walking through the file system, the listing is only for informational purposes
+		_ = filepath.Walk(flagValues.target, func(path string, info fs.FileInfo, err error) error {
+			if err != nil {
+				log.Printf("ignoring error for path %s: %v\n", path, err)
+				return nil
+			}
+
+			// Omit the target path itself
+			if path == flagValues.target {
+				return nil
+			}
+
+			// use relative paths to keep output compact
+			if l, err := filepath.Rel(flagValues.target, path); err == nil {
+				path = l
+			}
+
+			// if possible, try to obtain nlink count and user/group details
+			var nlink, user, group string = "?", "?", "?"
+			if stat, ok := info.Sys().(*syscall.Stat_t); ok {
+				user = strconv.FormatUint(uint64(stat.Uid), 10)
+				group = strconv.FormatUint(uint64(stat.Gid), 10)
+				nlink = strconv.FormatUint(uint64(stat.Nlink), 10)
+			}
+
+			t.Append([]string{filemode(info), nlink, user, group, strconv.FormatInt(info.Size(), 10), path})
```

Comment:
Do we want to use some library to create a human readable size ?

Reply:
There is one, and I weight it back and forth, same for the table writer and for the permission bits. What I can also do is to reuse the [human readable size function](https://github.com/homeport/havener/blob/db6796a9f2ff2eb86e43f451d06f59df14ed5551/internal/cmd/top.go#L571-L582) that I wrote for `havener`, it shouldn't add to much.

Reply:
Yeah, thought about having an own small and simple function as well. I suggest to go for it. We probably wont have the largest numbers in our context, but even on six vs seven digits numbers it imo helps. I suggest to use "B" instead of "Byte" for mods[0] as it would otherwise print "67 Byte" lacking the trailing "s".